### DISCUSSION TITLE
User Persona

### DISCUSSION THREAD

**Discussion by hitomipupil** [HYF-Class22/agile-development-group3-hyf-redesign#1] [General]
Let's talk about user persona here

**Comment by hitomipupil** [comment]
# Future student
*Prospective students who are willing to start their journey of learning web development from scratch.*

- **Bio**: Haftu graduated from University but struglling to find a job with his degree in literature. He decided to change his field to tech and eager to find a school and now comparing the school by looking at the websites.

- **Needs/Goals**: Haftu wants to choose one coding school based on the class atmosphere and how easy to contact

- **Frustrations**: He wants to know how is the atmosphere of the class by seeing some testimonies and photos. He also wants to contact the school to ask some questions but he cannot find the one explaining those points clearly.

- **Scenario 1**: Haftu is 25 years old graduate and looking for a coding school. He wants to choose one coding school based on the class atmosphere and how easy to contact with.

**Comment by hitomipupil** [comment]
Support

People who are interested in donating for the tech school

**Reply by rugiraprince** [reply]
@hitomipupil 
 user persona(support)
https://www.semrush.com/persona/share/LuiklKLEWMLEBcGpE37JroJ1X0I53qpBrzFKPVfzt3k/

**Reply by hitomipupil** [reply]
@rugiraprince 
Thank you! Could you also write a Scenario?
And I can't clearly understand the Frustration. Could you make it more clear and more related to website? I think she should have some frustrations by looking at the other web pages.

**Reply by rugiraprince** [reply]
@hitomipupil  okay am gonna do it

**Reply by rugiraprince** [reply]
https://www.semrush.com/persona/share/bPOfy1NiiFouXrS-WuIMWehoth6MbT3uNCuWMJcbDY4/

**Reply by hitomipupil** [reply]
@rugiraprince Thank you for the modification! For me the scenario is good :)
About Frustrations, I have one question: what is the "self-service" ??

**Reply by rugiraprince** [reply]
@hitomipupil is services where companies  provide a hub of product information where customers can search for answers to commonly asked questions on their own time, without talking to a customer service

**Reply by hitomipupil** [reply]
@rugiraprince Thank you for the explanation, I understand that is it about FAQ!

**Comment by hitomipupil** [comment]
Volunteer

Are Expertise in web development technologies and tools looking a better place to deliver their knowledge more in a structured way like schools such as HYF

**Reply by jgchoti** [reply]
**Bio:**  Hitomi Yamamoto is 32 years old with 7 years of experience in web development. Originally from Japan, she relocated to Belgium 5 years ago, where she has become an active member of the local tech community. Hitomi is passionate about making a positive impact on the local community through her expertise in web development.

**Needs/Goals:** Hitomi wants to be part of an environment that actively promotes equal opportunities for everyone who interested in learning web development. 

**Frustrations:** Hitomi is frustrated by the difficulty in finding testimonials and photos to understand the atmosphere of teaching environments. Hitomi also find that it is difficult to contact schools for program inquiries due to a lack of clear points of contact,  making it challenging to decide on volunteering opportunities. Hitomi is also frustrated when organizations don't have a responsive website. 

**Scenario 1:** Hitomi wants to volunteer with an organisation that aligns with her values. She wants to see a clear curriculum for the class and she is finding information about the teaching environment, including testimonials and photos. Hitomi also places importance on organizations that feature a responsive website.

@hitomipupil :)

**Reply by hitomipupil** [reply]
@jgchoti XDXDXD
Perfect!

**Comment by hitomipupil** [comment]
**Partnership**
Companies who are interested in hiring graduates from HYF.

**Bio:** `John ` is a big company owner interested to collaborate with schools offering market oriented specialties and would like to hire graduates with the right qualifications.
**Needs/Goals:** `John `is searching for more information on how to increase the tangible and intangible values of his company through collaboration with the schools by offering internships.
**Frustrations:** `John`  can't find a website which provides clear and concise user information on how to collaborate with schools that promote in producing talented graduates. Most websites offer complex information about collaboration and are too general and do not focus on the practicability of collaboration with the right schools
**Scenario 1:** `John `is a company owner who wants to have partnership with schools and and as a result, wants to hire talented graduates that are familiar to the job requirements in his company.

**Reply by hitomipupil** [reply]
@Tesfay20 Nice job!!
